.github/workflows/createTestList.py

```diff
@@ -0,0 +1,22 @@
+import sys
+import os
+import ast
+
+def main():
+    
+    sanity_format = "sanity.{}"
+    formattedTests = []
+    
+    targetNames = ast.literal_eval(os.getenv('TARGET_LIST'))
+    
+    for argument in targetNames:
+        formattedTests.append(sanity_format.format(argument))
+    
+    allTests = ','.join(formattedTests)    
+    testTargets = 'TESTLIST={}'.format(allTests)
+    
+    print(testTargets)
+    print('::set-output name=test_targets_str::{}'.format(testTargets))
+
+if __name__ == "__main__":
+    main()
```

Comment:
This file may not need if we are setting the target as something like `sanity.openjdk`. We could simply set job 'trigger single job'  as params: `{"TARGET": "sanity", "BUILD_LIST": 'openjdk'}`

Reply:
`If both test dirs and buildenv, dir changes, can a Grinder be triggered with targets from the playlists within the test dirs.` That is instead of `sanity` we would like a testcase in playlist.xml. For example if build_list is `openjdk`, we can set target as `jdk_custom`. If build_list is `system` we could set target as `ClassLoadingTest`.

Reply:
@sophia-guo Hi Sophia! The issue seems to imply we're running all the targets in playlist.xml, but your example shows only one. Just wanted to double check that we're running all the tests in the playlist.xml file?

Reply:
Folders like perf also has subdirectories with their own playlist.xml files. 

If something in the perf folder changes, should we 
- Run tests from all the playlist.xml files in all the subfolders of perf? Regardless of which subdirectory changed (e.g. `perf/bumbleBench/playlist.xml`, `perf/dacapo /playlist.xml`, and so on)  
- Or just bring in targets from the the subdirectories that changed, as well as targets from perf/playlist.xml (the root folder) if one exists (e.g. if I only changed things in the `perf/dacapo` subdirectory, to only run `perf/dacapo /playlist.xml` and `perf/playlist.xml` if one exists)

Reply:
https://github.com/AdoptOpenJDK/openjdk-tests/pull/2292#discussion_r582749502 I think an easy test case for each category should be enough as it is to check if the grinder still work.  @ShelleyLambert your opinion?

Reply:
Yes, agree, a single test target is sufficient (no need to run a full sanity or extended suite).

Reply:
Updated the params!
![image](https://user-images.githubusercontent.com/30379111/109300588-8a862880-77eb-11eb-929b-ab97b80403ed.png)